src/librustc_mir/transform/type_check.rs

```diff
@@ -487,6 +487,16 @@ impl<'a, 'gcx, 'tcx> TypeChecker<'a, 'gcx, 'tcx> {
 
                 if self.is_box_free(func) {
                     self.check_box_free_inputs(mir, term, &sig, args);
+                    // THIS IS A TEST. TEST TEST.
```

Comment:
what does this comment mean?

Reply:
I wanted to see that lints worked before I actually used them to implement effect checking.

Reply:
this should never have reached github. removing.